### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ArsenSkandalenko**
https://arsenskandalenko.github.io/react_people-table-basics/#/people

### CODE REVIEW

File: src/components/PeopleTable.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people?.map(person => {
+          return (
+            <tr
+              data-cy="person"
```
Comment (volodymyr-soltys97) [inline src/components/PeopleTable.tsx:34]: Don't forget about the key attribute

File: src/components/PeopleTable.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+              className={person.slug === slug ? 'has-background-warning' : ''}
```
Comment (volodymyr-soltys97) [inline src/components/PeopleTable.tsx:35]: Use the classnames library for add class with condition

File: src/components/PeopleTable.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+              <td>{person.sex}</td>
+              <td>{person.born}</td>
+              <td>{person.died}</td>
+
+              {!allPeopleNames?.includes(person?.motherName!) ? (
+                <td>{person.motherName ? person.motherName : '-'}</td>
+              ) : (
+                <td>
+                  {/* For better readabity i separate it another <td> */}
```
Comment (volodymyr-soltys97) [inline src/components/PeopleTable.tsx:49]: Remove comments everywhere
```suggestion
```

File: src/pages/People.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            </p>
+          ) : !isLoading ? (
+            people?.length! > 0 ? (
+              <PeopleTable people={people} />
+            ) : (
+              <p data-cy="noPeopleMessage">There are no people on the server</p>
+            )
+          ) : (
+            <Loader />
+          )}
```
Comment (volodymyr-soltys97) [inline src/pages/People.tsx:50]: This condition looks quite complex and confusing, try breaking it down into several different conditions

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

File: src/components/PeopleTable.tsx (hunk 1 above)
Comment (anastasiiavorobiova) [reply src/components/PeopleTable.tsx:34]: Not fixed

File: src/components/PeopleTable.tsx (hunk 2 above)
Comment (anastasiiavorobiova) [reply src/components/PeopleTable.tsx:35]: Not fixed

File: src/components/PeopleTable.tsx (hunk 3 above)
Comment (anastasiiavorobiova) [reply src/components/PeopleTable.tsx:49]: Not fixed

File: src/pages/People.tsx (hunk 4 above)
Comment (anastasiiavorobiova) [reply src/pages/People.tsx:50]: Not fixed

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Consider fixing previous comments

File: src/components/PeopleTable.tsx (hunk 1 above)
Comment (anastasiiavorobiova) [inline src/components/PeopleTable.tsx:34]:
```suggestion
              data-cy="person"
              key={person.slug}
```

File: src/components/PeopleTable.tsx (hunk 2 above)
Comment (anastasiiavorobiova) [inline src/components/PeopleTable.tsx:35]: Consider using class names library for classes

File: src/components/PersoneLink.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+export interface IPersoneLink {
+  person: Person;
+}
+
+export const PersoneLink: FC<IPersoneLink> = ({ person }) => {
+  return (
+    <Link
+      to={`../${person?.slug}`}
+      className={person?.sex === 'f' ? 'has-text-danger' : ''}
```
Comment (anastasiiavorobiova) [inline src/components/PersoneLink.tsx:13]: The same issue here

File: src/pages/People.tsx (hunk 4 above)
Comment (anastasiiavorobiova) [reply src/pages/People.tsx:50]: Not fixed

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Feel free to ask for help in the chat

**Pull request closed without merging**